src/v2/view-builder/internals/FormInputFactory.js

```diff
@@ -116,7 +116,7 @@ const createIdpButtons = (remediations) => {
     if (!_.contains(IDP.SUPPORTED_SOCIAL_IDPS, type)) {
       type = 'general-idp';
       // OKTA-396684 - makes sure that custom idps always have a name
-      displayName = loc('customauth.sign.in.with.label', 'login') + ` ${idpObject.idp?.name}`;
+      displayName = loc('customauth.sign.in.with.label', 'login', [idpObject.idp?.name]);
```

Comment:
@jmelberg-okta i have updated this.. but i don't see translation. What am i missing ?
![Screen Shot 2021-06-08 at 2 49 18 PM](https://user-images.githubusercontent.com/38230587/121262373-dc1d6500-c868-11eb-8bc6-d02f200d8a6c.png)

Reply:
ok_PL was cached locally.. thanks @sherizada-okta for the pointer. Works now.